### DISCUSSION TITLE
Export to xlsx

### DISCUSSION THREAD

**Discussion by BernardoCorreia** [Power-Components/livewire-powergrid#1702] [Q&A]
Hello! When exporting to excel numbers are exported as string. Any one knows how to export numbers as numbers?

Thanks

**Comment by luanfreitasdev** [comment]
Hello @BernardoCorreia !

Can you share more details, such as how it is in the database, how you set up the table, and how it was exported?

**Reply by BernardoCorreia** [reply]
Hello Luan. I have tried your demo https://demo.livewire-powergrid.com/examples/export and the id is also exported as text.

Can you check?

Thanks,
Bernardo Correia
![luandemo](https://github.com/user-attachments/assets/3861a5f4-5bfe-4eab-a287-6da89740b3bb)
